.travis.yml

```diff
@@ -2,12 +2,27 @@ language: python
 cache: pip
 dist: xenial
 
-matrix:
+jobs:
   include:
     - name: "shell syntax"
+      stage: tests
       script: find . -path "./submodules" -prune -o -type f -name "*.sh" -print | xargs -n1 shellcheck -x --format=gcc
     - name: "playbook syntax"
+      stage: tests
       script: ansible-lint playbooks/*.yml
+    - name: "build and publish docs"
+      stage: publish docs
+      install:
+        - pip install tox
+      script: tox
```

Comment:
can you change this to tox -e docs ?
Tox might grow with reno and linting.

Reply:
:+1: